### PR TITLE
DuHoot Website

### PR DESCRIPTION
**Pull request by editperson**
Зверстана веб-сторінка (Adaptive & Responsive) за шаблоном №9 (див. папку source).
Приницип верстки: Mobile First

### CODE REVIEW

**Comment by editperson** [comment]
http://duhoot.robik-music.com/ - link to website

File: css/style.css [hunk 1]
```diff
@@ -1,44 +1,372 @@
...
+         url('fonts/GothamMedium.ttf') format('truetype'),
+         url('fonts/GothamMedium.svg#GothamMedium') format('svg');
 }
 
 body{
-	font-family: "Gotham" sans-serif;
+	font-family: "GothamBook", sans-serif;
 }
 
 		/*Тут буде писати свої стилі Ліда*/
```
Comment (tornado05) [inline css/style.css]: this should be removed

File: css/style.css [hunk 2]
```diff
@@ -1,44 +1,372 @@
...
 
 /* Custom, iPhone Retina */
 @media only screen and (min-width : 320px) {
+	header {
+		width:100%;
+	}
+
+	.container {
+		max-width: 980px;
```
Comment (tornado05) [inline css/style.css]: this should be moved to media query

File: css/style.css [hunk 3]
```diff
@@ -1,44 +1,372 @@
...
+		color: #B9B9B9;
+	    text-transform: uppercase;
+	    display: inline-block;
+	    font-family: 'GothamMedium';
+	    font-size: 14px;
+	    padding: 10px 5px 20px 10px;
+	}
+
+	.menu ul li:hover {
+		 -webkit-transition: all .5s;
```
Comment (tornado05) [inline css/style.css]: FORMATTING!!!!!!

File: css/style.css [hunk 4]
```diff
@@ -1,44 +1,372 @@
...
+	     border: none;
+	     -webkit-transition: all .3s;
+	     -moz-transition: all .3s;
+		 transition: all .3s;
+		 padding:10px;
+		 font-size: 16px;
+	}
+
+	input[type=search]:focus {
+    width: 237px;
```
Comment (tornado05) [inline css/style.css]: FORMATTING!!!!!!

File: css/style.css [hunk 5]
```diff
@@ -1,44 +1,372 @@
...
+
+	.form-select select {
+    width: 94%;
+	}
 
 }
 
 /* Medium Devices, Desktops */
-@media only screen and (min-width : 1024px) {
+@media only screen and (min-width : 992px) {
```
Comment (tornado05) [inline css/style.css]: why 992?

File: css/style.css [hunk 6]
```diff
@@ -89,53 +624,458 @@ body{
...
+		position: absolute;
+		top: 184px;
+		left: 0;
+		width: 100%;
+		
+	}
 }
 
  /* Large Devices, Wide Screens */
 @media only screen and (min-width : 1280px) {
```
Comment (tornado05) [inline css/style.css]: why 1280? big devices starts from 1200.

File: css/style.css [hunk 7]
```diff
@@ -76,10 +404,217 @@ body{
...
+	.tab .column .icon-bolt {
+		background: #1db05e url('../img/offer/bg_ico_bolt.png') 50% 50% no-repeat;
+	}
+
+	.tab .column .icon-leaf {
+		background: #1db05e url('../img/offer/bg_ico_leaf.png') 50% 50% no-repeat;
+	}
 
 /* Custom, iPhone Retina */
 @media only screen and (min-width : 320px) {
```
Comment (tornado05) [inline css/style.css]: you already have the same query [here](https://github.com/editperson/4etkost/pull/3/files#diff-6a6a912d0c7b55b537768da778032849L114), why do we have two of them?

File: css/style.css [hunk 8]
```diff
@@ -89,53 +624,458 @@ body{
...
+			 -moz-border-radius: 50%;
+		        border-radius: 50%;
+		-webkit-background-size: auto 45%;
+		     -o-background-size: auto 45%;
+		        background-size: auto 45%;
+	}
+
+	.tab label:before {
+		top: 50%;
+		margin-top: -1px;
```
Comment (tornado05) [inline css/style.css]: why -1?

File: css/style.css [hunk 9]
```diff
@@ -89,53 +624,458 @@ body{
...
+		left: -37px;
```
Comment (tornado05) [inline css/style.css]: the same?

File: css/style.css [hunk 10]
```diff
@@ -1,44 +1,372 @@
...
 @media only screen and (min-width : 480px) {
-	
+	.mid-header .title {
+		font-size: 22px;
+		font-weight: bold;
+	}
 }
 
 /* Small Devices, Tablets */
 @media only screen and (min-width : 768px) {
```
Comment (tornado05) [inline css/style.css]: you have two queries with `min-width : 768px` why?

**Comment by tornado05** [comment]
If you use CPDD, clean after and look what you're doing

File: css/style.css (hunk 8 above)
Comment (aklynovyy) [inline css/style.css]: For vertical align

File: css/style.css [hunk 11]
```diff
@@ -89,53 +624,458 @@ body{
...
+.contact1  {
+  padding-left: 5px;
+}
+.contact2 {
+padding-left: 5px;
+}
+.contact3  {
+  padding-left: 15px;
+}
+.contact3 img {
```
Comment (tornado05) [inline css/style.css]: what is the empty class doing here?

File: css/style.css [hunk 12]
```diff
@@ -89,53 +624,458 @@ body{
...
+
+  footer .social {
+	margin-bottom: 0;
+  }
+
+  .contact1 a, .contact2 a,.contact3 p {
+  line-height: 2;
+}
+
+  footer input::-webkit-input-placeholder {
```
Comment (tornado05) [inline css/style.css]: :moz-placeholder is deprecated [here](https://developer.mozilla.org/en-US/docs/Web/CSS/:-moz-placeholder), you should use `::-moz-placeholder`

File: css/style.css [hunk 13]
```diff
@@ -89,53 +624,458 @@ body{
...
+
+	.tab {
+		width: auto;
+	}
+
+	.tab label {
+		margin-left: 56px;
+		width: 104px;
+		height: 104px;
+		-webkit-border-radius: 50%;
```
Comment (tornado05) [inline css/style.css]: FORMATTING!!!

File: css/style.css [hunk 14]
```diff
@@ -89,53 +624,458 @@ body{
...
+		border: 0 none;
+	}
+
+	.tab label {
+		margin-left: 35px;
+		width: 75px;
+		height: 75px;
+		border: 2px solid #9dacad;
+		-webkit-border-radius: 50%;
+			 -moz-border-radius: 50%;
```
Comment (tornado05) [inline css/style.css]: FORMATTING!!!!

File: css/style.css [hunk 15]
```diff
@@ -1,44 +1,372 @@
...
+		display:inline-block;
+		color:#fff;
+		width:48%;
+		text-transform: uppercase;
+		font-size: 14px;
+		background:#e2dee0;
+		text-align: center;
+	}
 
+	input[type=search]{
```
Comment (tornado05) [inline css/style.css]: who stole the quotes?

File: css/reset.css [hunk 16]
```diff
@@ -6,9 +6,9 @@ figure, figcaption, footer, header, hgroup, menu, nav, output, section, summary,
   border: 0;
   font-size: 100%;
   font: inherit;
-  vertical-align: baseline;
+  vertical-align: top;
```
Comment (tornado05) [inline css/reset.css]: this should not be in reset css, here you should use `vertical-align: baseline`

File: index.html [hunk 17]
```diff
@@ -11,7 +11,63 @@
...
-
+	<header>
+	  <div class="container">
+		<div class="top-header">
+			<div class="logo">
+				<a href="#"><img src="img/header/logo.png"/></a>
+			</div>
+			<nav class="menu">
+				<ul>
+					<a href="#"><li>Reality</li></a>
```
Comment (tornado05) [inline index.html]: 'a' should be inside of `li`

**Comment by tornado05** [comment]
:+1:7

File: css/style.css (hunk 6 above)
Comment (editperson) [inline css/style.css]: Ми вже третій раз берем ці параметри на домашку... ще з часу роботи в команді з Дробенюком...

File: css/style.css (hunk 10 above)
Comment (editperson) [inline css/style.css]: what about you?

File: css/style.css (hunk 11 above)
Comment (editperson) [inline css/style.css]: CPDD - College on Problems of Drug Dependence? :-/

File: css/style.css (hunk 2 above)
Comment (editperson) [inline css/style.css]: It's not in the media query? Or what do you mean?

File: css/style.css (hunk 1 above)
Comment (editperson) [inline css/style.css]: from pull request? you mean that we should not have changed files in pull request?

Comment (tornado05) [inline css/style.css]: no, before you release your work make the clean up

File: css/style.css (hunk 2 above)
Comment (tornado05) [inline css/style.css]: you use `max-width` in query that should control 320-768px, and you set the value to 980?

File: css/style.css (hunk 8 above)
Comment (tornado05) [inline css/style.css]: try to avoid this

File: css/style.css (hunk 6 above)
Comment (tornado05) [inline css/style.css]: ok, last week we learned the grid, also in the task I was telling you 1200, so where did you get 12**80**?

File: css/style.css (hunk 10 above)
Comment (tornado05) [inline css/style.css]: what about me?! You have two queries with `min-width: 768px`, There should be one? nope?

File: css/style.css (hunk 11 above)
Comment (tornado05) [inline css/style.css]: Cope Paste Driven Development

File: css/style.css (hunk 1 above)
Comment (editperson) [inline css/style.css]: Andrew (during work) found a complete font format and we decided to replace.

File: css/style.css (hunk 2 above)
Comment (editperson) [inline css/style.css]: emmm.. I use min-width, not max-width... or what do you mean...?

File: css/style.css (hunk 1 above)
Comment (tornado05) [inline css/style.css]: I'm talking about the comments

File: css/style.css (hunk 10 above)
Comment (editperson) [inline css/style.css]: I don't see it. Where?... I see just one for each of the developers ...

**Pull request merged**